globalsSelectors.ipynb

```diff
@@ -40,7 +40,7 @@
    "source": [
     "#number of data per visualisation with x axis\n",
     "global xaxis\n",
```

Comment:
The current functionality of these two 'order' and 'xaxis' selectors makes that, every time their value is changed, that affects all visualizations. But they appear on each visualization that makes sense to have them, therefore, giving the impression that their are a selector local to that visualization. 

I suggest their value should only affect the visualization they are above to (as, for instance, I may want to see one visualization with values from highest to lowest as it indicates scores, but other visualization with player data I would prefer to see alphabetically).